internal/config/config.go

```diff
@@ -79,3 +80,7 @@ func (c Config) ResolveSharedCredentialsFiles() ([]string, error) {
 	}
 	return v, nil
 }
+
+func (c Config) GetIdentityToken() ([]byte, error) {
```

Comment:
I need it for https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/credentials@v1.10.0/stscreds#NewWebIdentityRoleProvider / https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/credentials@v1.10.0/stscreds#IdentityTokenRetriever

Let me know if you prefer to have 'implementation' of that interface separately from `Config` type

Reply:
Since it only references fields from `c.AssumeRoleWithWebIdentity`, it would probably be good to move it there